### PR TITLE
console: make error notifications dismissible only after clicking on the close button (close #5533)

### LINKED ISSUE #5533: Simple Click on error panel in console close it
Hello All,

Hasura v1.3.0
Console UI bug.

When I have an error when trying to rename a table, the error panel close when I click on it (for text selection).
Kind of annoying.

### PR DESCRIPTION
**Pull request by Varun-Choudhary**
Closes: #5533 

### Description

Changed the `Dismissible` property of `React-Notifiction-System` so that the error notification will disable by clicking on `X`  button only.

### Affected components 

- [X] Console

### CODE REVIEW

File: console/src/components/App/Actions.js [hunk 1]
```diff
@@ -16,7 +16,7 @@ const CONNECTION_FAILED = 'App/CONNECTION_FAILED';
  * title: string null
  * message: string null
  * autoDismiss: integer 5, set to 0 to not auto-dismiss
- * dismissible: bool true, set if user can dismiss notification
+ * dismissible: button, The user can click on the disable button to dismiss the notifiction
```
Comment (beerose) [inline console/src/components/App/Actions.js:19]:
```suggestion
 * dismissible: 'button', set if the user can dismiss the notification, false otherwise

**Review by beerose — APPROVED** [review]
Console

**Review by tirumaraiselvan — APPROVED** [review]
cl

File: console/src/components/Services/Common/Notification.tsx [hunk 2]
```diff
@@ -43,7 +43,9 @@ export const showNotification = (
         {
           position: options.position || 'tr',
           autoDismiss: ['error', 'warning'].includes(level) ? 0 : 5,
-          dismissible: ['error', 'warning'].includes(level),
+          dismissible: ['error', 'warning'].includes(level)
+            ? ('button' as any) // bug in @types/react-notification-system-redux types
+            : false,
```
Comment (beerose) [inline console/src/components/Services/Common/Notification.tsx:48]:
```suggestion
            : ('click' as any),
```

**Pull request merged**